tutorials/physics/using_kinematic_body_2d.rst

```diff
@@ -170,7 +177,42 @@ Attach a script to the KinematicBody2D and add the following code:
 
  .. code-tab:: csharp
 
-    Coming soon
+    using Godot;
+    using System;
+
+    public class KBExample : KinematicBody2D
+    { 
+        int speed = 250;
+        Vector2 velocity = new Vector2();
```

Comment:
Probably we should make `speed` a public property, and `velocity` a private field (with respective naming changes).

On a side note, I think declaring `velocity` as a member variable is sort of a bad practice. But I can see how it's duplicated from the GDScript counter part, so probably it needs to be dealt with a separate PR if we ever fix it.